src/Categories/Category/Construction/mu-Bialgebras.agda

```diff
@@ -0,0 +1,63 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Construction.mu-Bialgebras where
+
+open import Level
+open import Function using (_$_)
+
+open import Categories.Category
+open import Categories.Functor hiding (id)
+open import Categories.Functor.Algebra
+open import Categories.Functor.Coalgebra
+import Categories.Morphism.Reasoning as MR
+open import Categories.Functor.Bialgebra
+open import Categories.Category.Construction.F-Algebras
+open import Categories.Category.Construction.F-Coalgebras
+open import Categories.Functor.DistributiveLaw using (DistributiveLaw)
+
+private
+  variable
+    o ℓ e : Level
+
+μ-Bialgebras : {C : Category o ℓ e} → (T F : Endofunctor C) → (DistributiveLaw T F) → Category (o ⊔ ℓ ⊔ e) (o ⊔ ℓ ⊔ e) e
+μ-Bialgebras {C = C} T F μ = record
+ { Obj       = μ-Bialgebra T F μ
+ ; _⇒_       = μ-Bialgebra-Morphism
+ ; _≈_       = λ β₁ β₂ → f β₁ ≈ f β₂
+ ; _∘_       = λ β₁ β₂ → record
```

Comment:
It's probably a good idea to define `μ-Bialgebra-Morphism` composition where your define `μ-Bialgebra-Morphism` rather than inline here. Same with `id`. [Many other parts of the library do that already.] The rest can stay here.

Reply:
Ah, I was mostly inspired by `Algebra` in relation to `F-Algebras` (where composition is defined in the Category construction `F-Algebras`)

Reply:
Could you give a concrete example that I could use as reference?

Reply:
[Categories.Category.Construction.Comma](https://agda.github.io/agda-categories/Categories.Category.Construction.Comma.html) does it in a `where` block.  [Functors](https://agda.github.io/agda-categories/Categories.Category.Construction.Functors.html#1304) defines it elsewhere (but it's cheating a little bit).

I think [LawvereTheories](https://agda.github.io/agda-categories/Categories.Category.Instance.LawvereTheories.html) is a good example of the 'separate' style.